### PR TITLE
chore(ls): linting cleanup

### PR DESCRIPTION
**Pull request by Ivan01925**
Hi! how I mentioned in the issue #3453, there are variables that are never used in file _ls.js_ and ESLint error in file _npx-cli.js_.

I made the correction and I ran the tests, there were no problems:

image

## References
Issue: #3453

### CODE REVIEW

**Comment by wraithgar** [comment]
We should fix the root cause of this problem, namely that `bin` is not in the list of folders that are linted.  Can you add that to the `lint` entry in `package.json`?

**Comment by Ivan01925** [comment]
> We should fix the root cause of this problem, namely that `bin` is not in the list of folders that are linted. Can you add that to the `lint` entry in `package.json`?

Done, now is also checking bin folder :)

**Review by darcyclarke — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -212,7 +212,7 @@
     "sudotest:nocleanup": "sudo NO_TEST_CLEANUP=1 npm run test --",
     "posttest": "npm run lint",
     "eslint": "eslint",
-    "lint": "npm run eslint -- test/lib test/bin lib scripts docs smoke-tests",
+    "lint": "npm run eslint -- test/lib test/bin ./bin lib scripts docs smoke-tests",
```
Comment (wraithgar) [inline package.json]:
```suggestion
    "lint": "npm run eslint -- test/lib test/bin bin lib scripts docs smoke-tests",
```

**Comment by wraithgar** [comment]
Thanks! Letting the CI run now to be sure nothing is missed now that `bin` is linting.

**Pull request merged**